.travis.yml

```diff
@@ -127,30 +199,23 @@ install:
   # Implement a platform-independent timeout function.
   function timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
 
+  # If we timeout, objects can be corrupted in the cache, causing link errors
+  # Turn this on to get the mapping to the cache.o to delete
+  # export CCACHE_LOGFILE="$CLING_USER_ROOT/ccache.log"
+
   if [ ! -f $CMAKE ]; then
     mkdir -pv $CLING_BUILD_DEPS/cmake
     travis_retry curl -k $CMAKE_URL | tar --strip-components=1 -xz -C $CLING_BUILD_DEPS/cmake
   fi
 
 before_script:
-- |
-  # Get the real paths of the compilers, before overriding it.
-  REAL_CXX=$(which $CXX_COMPILER$COMPILER_VERSION)
-  REAL_CC=$(which $CC_COMPILER$COMPILER_VERSION)
-
-  echo "Making $CXX_COMPILER$COMPILER_VERSION and $CC_COMPILER$COMPILER_VERSION default."
-  mkdir -pv $CLING_LOCAL_BIN
-
-  echo -e "#!/bin/bash\nccache $REAL_CXX \"\$@\"" > $CLING_LOCAL_BIN/$CXX_COMPILER
-  echo -e "#!/bin/bash\nccache $REAL_CC \"\$@\"" > $CLING_LOCAL_BIN/$CC_COMPILER
-
-  chmod +x $CLING_LOCAL_BIN/$CXX_COMPILER
-  chmod +x $CLING_LOCAL_BIN/$CC_COMPILER
-
-- export PATH=$CLING_LOCAL_BIN:$PATH
-- eval $EXECUTE_COMMAND
+ - $CMAKE --version
+ - export CXX="ccache $COMPILER"
+ - export CC="ccache $CCOMPILER"
 
 script:
+- $CXX --version
```

Comment:
I'm gonna cancel this pull request....
I think there's a way to get travis to use the cache I already have rather than rebuilding.